packages/fast-tooling-react/src/css-editor/editor.tsx

```diff
@@ -9,6 +9,8 @@ import { CSSPosition, CSSPositionValues, Location, PositionValue } from "./posit
 import { CSSSpacing, CSSSpacingValues, SpacingProperty } from "./spacing";
 import { CSSWidth, CSSWidthValues } from "./width";
 import { CSSHeight, CSSHeightValues } from "./height";
+import { CSSColor, CSSColorValues } from "./color";
+import { CSSBorder, CSSBorderValues } from "./border";
```

Comment:
did this come from another PR? I don't see the border files here.

Reply:
good catch yeah I built them together and then separated them out into 2 PRs this must have been left over